### DISCUSSION TITLE
Preview image different than "Open link" url

### DISCUSSION THREAD

**Discussion by TorgeirHeim** [mozilla/hubs#3437] [General Questions & Discussion] [answered]
Is it possible to show a preview image when adding a link in hubs and have it open a different link when pressed?

**Comment by cvanmeurs** [comment]
I believe the preview image uses the same image that the "scene preview camera" uses in Spoke for the room thumbnail.  You can set that wherever you want.  

If you're talking about a web URL, then I dunno what it will show - probably depends on how the website in particular is set up.

What are you trying to do exactly?

**Reply by TorgeirHeim** [reply]
We're trying to set up a virtual Christmas market where vendors can display and sell their merchandise by linking to different webshops. The problem is that when adding a webshop link in hubs the webpage is displayed. We would like to show an image of the product. 

I've set up a proxy server to redirect to the webshop when the request comes from "Open link" and to the image otherwise by looking at the http headers in the different cases. This is error prone and breaks for certain images. It would be convenient to have hubs display preview images for webpages in the same way as Open Graph images.

**Reply by robinkwilson** [reply]
**This could be a feature request for spoke. Make an issue/feature request here: https://github.com/mozilla/Spoke/issues**

**Comment by rawnsley** [comment, marked as answer]
Workaround proposed here: https://github.com/mozilla/hubs/discussions/3080

**Reply by robinkwilson** [reply]
Marking this as answer. Please link the feature request if you create one in https://github.com/mozilla/Spoke/issues
